admin/debian/control

```diff
@@ -0,0 +1,12 @@
+Source: securedrop-admin
+Section: web
+Priority: optional
+Maintainer: SecureDrop Team <securedrop@freedom.press>
+Build-Depends: debhelper (>= 11), dh-python, python3-all, python3-setuptools
+Homepage: https://securedrop.org
+Standards-Version: 4.5.1
+
+Package: securedrop-admin
+Architecture: amd64
+Depends: python3-virtualenv, python3-yaml, python3-pip, libffi-dev, libssl-dev, libpython3-dev, sq-keyring-linter, netcat-openbsd, tor, rsync
```

Comment:
I don't think we need most of these python- dependencies at runtime, the e.g. virtualenv has already been created, we shouldn't be pip installing anything. Same with the -dev ones, since we're not compiling anything.